src/testapp/scenarios.cpp

```diff
@@ -370,6 +370,101 @@ static ScenarioMap_t make_scenarios()
         return setup_renderer;
     });
 
+    add_scenario("lander", "Lander simulation game", [] (TestApp& rTestApp) -> RendererSetupFunc_t {
+        #define SCENE_SESSIONS scene, commonScene, uniCore, uniScnFrame, uniPlanet, physics, \
+            prefabs, parts, signalsFloat, vehicleSpawn, vehicleSpawnVB, vehicles, \
+            newton, vehicleSpawnNwt, nwtRocketSet, rocketsNwt, \
+            machRocket, machRcsDriver
+        #define SCENE_SESSIONS_COUNT 18
+        #define RENDERER_SESSIONS sceneRenderer, magnumScene, planetDraw, \
+            cameraCtrl, cameraFree, shVisual, shFlat, shPhong, \
+            prefabDraw, vehicleDraw, vehicleCtrl, cameraVehicle
+        #define RENDERER_SESSIONS_COUNT 12
+
+        using namespace testapp::scenes;
+
+        auto const  defaultPkg      = rTestApp.m_defaultPkg;
+        auto const application     = rTestApp.m_application;
+        auto & rTopData            = rTestApp.m_topData;
```

Comment:
minor: whitespace alignment.

Reply:
It's a little rough maintaining style without automation to do it.
For now while working on the PR I'll rely on automated formatting of my editor to not go insane. If needed I'll update formatting when this is somewhat ready.